### DISCUSSION TITLE
How to get remote input working on Ubuntu 23.04

### DISCUSSION THREAD

**Discussion by Humorist2601** [andyholmes/gnome-shell-extension-gsconnect#1603] [Q&A]
### Describe the bug

Hello, I am currently using Ubuntu 23.04 Development preview and want to know how can I get this feature working.

I know you said that all ubuntu version using wayland dont support this feature but there might be something we can install or change that makes the feature available on Ubuntu. Thanks :)

### Steps to reproduce

1. Go to GSConnect
2. No remote input available

### Expected behavior

Remote input should be available

### GSConnect version

GSConnect 54

### Installed from

GNOME Extensions website

### GNOME Shell version

GNOME 43.2

### Linux distribution/release

Ubuntu 23.04

**Comment by daniellandau** [comment]
What? Just tested with Ubuntu 22.10 and GNOME 43.1 on wayland and remote input of both keyboard and mouse in both directions work.

**Comment by Humorist2601** [comment]
It works on Xorg but not on Wayland. I searched and found out you need GNOME Remote Desktop for it to work. I installed it but it still doesnt work. Fwiw I used the minimal install of ubuntu

**Comment by Humorist2601** [comment]
![Screenshot from 2023-02-18 21-32-14](https://user-images.githubusercontent.com/97790626/219877245-c7a132eb-005d-4343-a386-a6001a750d0b.png)
This is my remote desktop settings. It seems to be missing the Remote Desktop Address

**Comment by daniellandau** [comment]
If you just installed the remote desktop support maybe try rebooting?

**Comment by ferdnyc** [comment]
Converting this to a Discussion, as it doesn't currently report any sort of fixable GSConnect bug.
